Fomo3Dlong.sol

```diff
@@ -1352,16 +1352,13 @@ contract FoMo3Dlong is modularLong {
         uint256 _long = _eth / 100;
         otherF3D_.potSwap.value(_long)();
         
-        // distribute share to affiliate
-        uint256 _aff = _eth / 10;
-        
         // decide what to do with affiliate share of fees
         // affiliate must not be self, and must have a name registered
         if (_affID != _pID && plyr_[_affID].name != '') {
-            plyr_[_affID].aff = _aff.add(plyr_[_affID].aff);
-            emit F3Devents.onAffiliatePayout(_affID, plyr_[_affID].addr, plyr_[_affID].name, _rID, _pID, _aff, now);
+            f3DReferral(_rID, _pID, _eth, _affID);
         } else {
-            _p3d = _aff;
+            //if there is no affiliate, _p3d = 10% tokens
+            _p3d = _eth/ 10;
```

Comment:
if there is no affiliate code given,i am assigning entire 10% to _p3d. is that ok ?